content/scout/policy/ci.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Evaluate policy compliance in CI
+description: |
+  Configure your continuous integration pipelines to fail
+  when Policy Evaluation for an image is worse compared to baseline
+keywords: scout, supply chain, policy, ci
+---
+
+Adding Policy Evaluation to your continuous integration pipelines helps you
+detect and prevent cases where code changes would cause policy status become
+worse compared to your baseline.
+
+The recommended strategy for Policy Evaluation in a CI setting involves
+evaluating a local image and comparing the results to a baseline. If the policy
+compliance for the local image is worse than the specified baseline, the
+CI run fails with an error. If the policy compliance is unchanged or improved,
+the CI run succeeds.
+
+This comparison is relative, meaning that it's only concerned with whether your
+CI image is better or worse than your baseline. It's not an absolute check to
+pass or fail all policies. By measuring relative to a baseline that you define,
+you can quickly see if a change has a positive or negative impact on policy
+compliance.
+
+## How it works
+
+When you do Policy Evaluation in CI, you run a local policy evaluation on the
+image you build in your CI pipeline. To run a local evaluation, the image that
+you evaluate must exist in the image store where your CI workflow is being run.
+Either build or pull the image, and then run the evaluation.
+
+To run policy evaluation and trigger failure if compliance for your local image
+is worse than your comparison baseline, you need to specify the image version
+to use as a baseline. You can hard-code a specific image reference, but a
+better solution is to use [environments](../integrations/environment/_index.md)
+to automatically infer the image version from an environment. The examples that
```

Comment:
I'm only seeing one example below. Can we edit "The examples that follow..." (plural) >>> "The example that follows..." (singular)?

Reply:
yep, must've been from when I was aiming to have multiple examples (but difficult to show due to how local policy evaluation is done)